v0/pipeline.json

```diff
@@ -19234,14 +19242,14 @@
               }
             }
           },
-          "AnchoreScanNode" : {
-            "title" : "AnchoreScanNode",
+          "DBApplySchemaStepNode" : {
```

Comment:
is this just a git diff issue that `AnchoreScanNode` is removed ?

Reply:
May be it was removed by someone or someone updated json. I have run script to generate it and not updated manually. 
Checking reference of this AnchoreScanNode.

Reply:
I think it is git diff only. I see some of the properties marked removed present in json.